pom.xml

```diff
@@ -162,8 +162,8 @@
     <resteasy.version>3.0.8.Final</resteasy.version>
     <rs-api.version>2.0</rs-api.version>
     <scala2.11.version>2.11.12</scala2.11.version>
-    <!-- Please keep consistent with the one in Spark 3 -->
-    <scala2.12.version>2.12.10</scala2.12.version>
+    <!-- Please keep consistent with the one in Spark 3.2.2 -->
```

Comment:
Let' update spark3.version in this file as well

Reply:
Done